CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# Release History
+
+## 1.0.0 (2024-03-27)
+
+### Breaking Changes
```

Comment:
Let's remove "Breaking Changes" heading. I don't think it's a breaking change for our consumers (no changes required on their side)